elasticapm/base.py

```diff
@@ -456,6 +462,8 @@ def set_transaction_extra_data(self, data, _key=None):
 
     def close(self):
         self._traces_collect()
+        if self._send_timer:
```

Comment:
Is it possible that a new transaction is started during the time that `close` is processing? If so, it looks like it could happen that the `end_transaction` could trigger a new `_start_send_timer`. This is probably an edge case that shouldn't be too bad though, because I assume when `close()` is called the whole application is going to be restarted..

Reply:
exactly. Outside of test scenarios, `.close()` is only called by application shutdown logic. At this point, the application shouldn't accept any new transactions.